spring-data-relational/src/main/java/org/springframework/data/relational/core/mapping/DerivedSqlIdentifier.java

```diff
@@ -65,8 +65,9 @@ public String toSql(IdentifierProcessing processing) {
 	}
 
 	@Override
+	@Deprecated(since="3.0.5", forRemoval = false)
```

Comment:
We should deprecate that method with the next feature release, not with a bugfix. That would be `since="3.1"`

Reply:
And it should be `forRemoval = true` since we want to drop it eventually.